examples/tutorials/alpha-zero-tic-tac-toe/main.py

```diff
@@ -0,0 +1,56 @@
+import logging
+
+from Coach import Coach
+from DeterminedShim import shim
+from tictactoe.keras.NNet import NNetWrapper as nn
+from tictactoe.TicTacToeGame import TicTacToeGame as Game
+from utils import *
+
+log = logging.getLogger(__name__)
+
+args = dotdict(
+    shim.override_params(
+        {
+            "numIters": shim.max_length(5),
+            "numEps": 25,  # Number of complete self-play games to simulate during a new iteration.
+            "tempThreshold": 15,  #
+            "updateThreshold": 0.6,  # During arena playoff, new neural net will be accepted if threshold or more of games are won.
+            "maxlenOfQueue": 200000,  # Number of game examples to train the neural networks.
+            "numMCTSSims": 50,  # Number of games moves for MCTS to simulate.
+            "arenaCompare": 50,  # Number of games to play during arena play to determine if new net will be accepted.
```

Comment:
does this always override the value set in the config?

Reply:
`shim.override_params` will take a dictionary of defaults and return the modified dictionary with any determined values that are given. 
The order of precedence is 1) Trial Config 2) User Data 3) Default

`numIters` is a little different in that we also prime the default with a searcher value if it exists.
The order of precedence for that variable would be 1) Trial Config 2) User Data 3) Searcher Config 4) Default

Reply:
got it, so something set in the determined experiment config (like `arenaCompare`) would take precedence?